tests/testdata/regionalisation_data/GetResgionalizationData.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May  9 20:11:23 2019
+
+@author: ets
+"""
+
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May  9 11:24:45 2019
+
+@author: ets
+"""
+
+
+from birdy import WPSClient
+import pandas as pd
+#import numpy as np
+#import os
+import time
+
+url = "http://localhost:9099/wps"
+wps = WPSClient(url, progress=True)
+
+    
+nbBasin=5797
+
+for i in range(1, nbBasin+1):
```

Comment:
matlab habits die hard ?

Reply:
I had to preallocate my values first so it would work with the "RunSuccess" Flag. Not sure it's useful anymore. And yes, I still do 95% of my coding in Matlab 😄